source/beamline/PGM/PGMMonoGratingSelectionControl.cpp

```diff
@@ -77,24 +75,33 @@ void PGMMonoGratingSelectionControl::updateMoving()
 
 AMAction3 *PGMMonoGratingSelectionControl::createMoveAction(double setpoint)
 {
-	return AMActionSupport::buildControlMoveAction(gratingSelectionPVControl_, setpoint, false, false);
+	AMAction3 *result = 0;
+
+	if (setpoint == Low)
+		result = AMActionSupport::buildControlMoveAction(gratingSelectionPVControl_, 3);
+	else if (setpoint == Medium)
+		result = AMActionSupport::buildControlMoveAction(gratingSelectionPVControl_, 2);
+	else if (setpoint == High)
+		result = AMActionSupport::buildControlMoveAction(gratingSelectionPVControl_, 1);
```

Comment:
Just want to confirm that the mapping of setpoints of the gratingSelectionPVControl_ are correct:
Low (0)      --- 3
Medium (1) --- 2
High (2)      --- 1